apps/desktop/src/lib/pr/StackingPullRequestCard.svelte

```diff
@@ -34,39 +37,44 @@
 	const baseBranchService = getContext(BaseBranchService);
 	const project = getContext(Project);
 
+	const gitHost = getGitHost();
 	const gitHostListingService = getGitHostListingService();
-	const prStore = $derived($gitHostListingService?.prs);
-	const prs = $derived(prStore ? $prStore : undefined);
-
-	const listedPr = $derived(prs?.find((pr) => pr.sourceBranch === upstreamName));
-	const prNumber = $derived(listedPr?.number);
 
 	const prService = getGitHostPrService();
-	const prMonitor = $derived(prNumber ? $prService?.prMonitor(prNumber) : undefined);
 
-	const checksMonitor = getGitHostChecksMonitor();
+	const checksMonitor = $derived(sourceBranch ? $gitHost?.checksMonitor(sourceBranch) : undefined);
+	$effect(() => {
+		console.log(checksMonitor);
+	});
 	// This PR has been loaded on demand, and contains more details than the version
 	// obtained when listing them.
-	const pr = $derived(prMonitor?.pr);
-	const checks = $derived($checksMonitor?.status);
+	const checks = $derived(checksMonitor?.status);
 
 	// While the pr monitor is set to fetch updates by interval, we want
 	// frequent updates while checks are running.
-	$effect(() => {
-		if ($checks) prMonitor?.refresh();
-	});
+	// $effect(() => {
+	// 	if ($checks) prMonitor?.refresh();
+	// });
 
 	let isMerging = $state(false);
 
 	const lastFetch = $derived(prMonitor?.lastFetch);
 	const timeAgo = $derived($lastFetch ? createTimeAgoStore($lastFetch) : undefined);
 
 	const mrLoading = $derived(prMonitor?.loading);
-	const checksLoading = $derived($checksMonitor?.loading);
+	const checksLoading = $derived(checksMonitor?.loading);
 
-	const checksError = $derived($checksMonitor?.error);
+	$effect(() => {
+		console.log($checksLoading);
+	});
```

Comment:
Just wanted to note that there's an `$inspect` rune for this now (see: https://svelte-5-preview.vercel.app/docs/runes#$inspect)